project.godot

```diff
@@ -22,9 +22,17 @@ _global_script_class_icons={
 
 config/name="Flappy Race"
 config/description="Multiplayer flappy bird"
-run/main_scene="res://scenes/World.tscn"
+run/main_scene="res://scenes/Lobby.tscn"
 config/icon="res://icon.png"
 
+[autoload]
+
+Net="*res://scripts/network.gd"
```

Comment:
I'm obviously not familiar with the language but...
It looks like the Network instance is loaded as a singleton here, which wouldn't allow you to have more than 1 game active at any time. 

Seems more sensible to have a Controller singleton which would handle registering and unregistering a new instance of network each time a game is created

Reply:
You can probably ignore this comment. Just realised the instance of the game will be running on an individuals device, so it's probably fine being a singleton!

Reply:
A feature we'd probably want in the future would be a centralised server, where players can choose lobbies. If we were to do that, I think this becomes a good point!